03-wordemb-pytorch/wordemb-cbow.py

```diff
@@ -0,0 +1,122 @@
+from collections import defaultdict
+import math
+import time
+import random
+import torch
+
+
+class CBoW(torch.nn.Module):
```

Comment:
Maybe the change the name to `WordEmbCbow` to make it more specific.

Reply:
Done